tests/workflows/test_from_csv_to_parquet.py

```diff
@@ -0,0 +1,140 @@
+    ("FractionDate", "Float64"),
+    ("Actor1Code", "string[pyarrow]"),
+    ("Actor1Name", "string[pyarrow]"),
+    ("Actor1CountryCode", "string[pyarrow]"),
+    ("Actor1KnownGroupCode", "string[pyarrow]"),
+    ("Actor1EthnicCode", "string[pyarrow]"),
+    ("Actor1Religion1Code", "string[pyarrow]"),
+    ("Actor1Religion2Code", "string[pyarrow]"),
+    ("Actor1Type1Code", "string[pyarrow]"),
+    ("Actor1Type2Code", "string[pyarrow]"),
+    ("Actor1Type3Code", "string[pyarrow]"),
+    ("Actor2Code", "string[pyarrow]"),
+    ("Actor2Name", "string[pyarrow]"),
+    ("Actor2CountryCode", "string[pyarrow]"),
+    ("Actor2KnownGroupCode", "string[pyarrow]"),
+    ("Actor2EthnicCode", "string[pyarrow]"),
+    ("Actor2Religion1Code", "string[pyarrow]"),
+    ("Actor2Religion2Code", "string[pyarrow]"),
+    ("Actor2Type1Code", "string[pyarrow]"),
+    ("Actor2Type2Code", "string[pyarrow]"),
+    ("Actor2Type3Code", "string[pyarrow]"),
+    ("IsRootEvent", "Int64"),
+    ("EventCode", "string[pyarrow]"),
+    ("EventBaseCode", "string[pyarrow]"),
+    ("EventRootCode", "string[pyarrow]"),
+    ("QuadClass", "Int64"),
+    ("GoldsteinScale", "Float64"),
+    ("NumMentions", "Int64"),
+    ("NumSources", "Int64"),
+    ("NumArticles", "Int64"),
+    ("AvgTone", "Float64"),
+    ("Actor1Geo_Type", "Int64"),
+    ("Actor1Geo_Fullname", "string[pyarrow]"),
+    ("Actor1Geo_CountryCode", "string[pyarrow]"),
+    ("Actor1Geo_ADM1Code", "string[pyarrow]"),
+    ("Actor1Geo_Lat", "Float64"),
+    ("Actor1Geo_Long", "Float64"),
+    ("Actor1Geo_FeatureID", "string[pyarrow]"),
+    ("Actor2Geo_Type", "Int64"),
+    ("Actor2Geo_Fullname", "string[pyarrow]"),
+    ("Actor2Geo_CountryCode", "string[pyarrow]"),
+    ("Actor2Geo_ADM1Code", "string[pyarrow]"),
+    ("Actor2Geo_Lat", "Float64"),
+    ("Actor2Geo_Long", "Float64"),
+    ("Actor2Geo_FeatureID", "string[pyarrow]"),
+    ("ActionGeo_Type", "Int64"),
+    ("ActionGeo_Fullname", "string[pyarrow]"),
+    ("ActionGeo_CountryCode", "string[pyarrow]"),
+    ("ActionGeo_ADM1Code", "string[pyarrow]"),
+    ("ActionGeo_Lat", "Float64"),
+    ("ActionGeo_Long", "Float64"),
+    ("ActionGeo_FeatureID", "string[pyarrow]"),
+    ("DATEADDED", "Int64"),
+    ("SOURCEURL", "string[pyarrow]"),
+])
+
+
+def test_from_csv_to_parquet(from_csv_to_parquet_client, s3_factory, s3_url):
+    client = from_csv_to_parquet_client
+
+    s3 = s3_factory(anon=True)
+    files = s3.ls("s3://gdelt-open-data/events/")[-120:]
+    files = [f"s3://{f}" for f in files]
+
+    def download(path):
+        try:
+            df = dd.read_csv(
+                path,
+                sep='\t',
+                names=COLUMNSV1.keys(),
+                dtype=COLUMNSV1,
+                storage_options=s3.storage_options,
+                on_bad_lines="skip"
+            )
+            # TODO: Want to raise ValueError right away, not
+            # inside the graph later which persist (or nothing) would cause.
+            df = df.compute()
```

Comment:
I don't completely understand why you're doing this but I suggest to either
* Use a `worker_client` explicitly, see https://distributed.dask.org/en/stable/task-launch.html
* Specify sync/threads scheduler explicilty.

The implicit behavior works but has quite a few shortcomings.